packages/framework/ai-collab/src/explicit-strategy/idGenerator.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert, oob } from "@fluidframework/core-utils/internal";
+import { Tree, NodeKind } from "@fluidframework/tree/internal";
+import type {
+	TreeNode,
+	ImplicitFieldSchema,
+	TreeArrayNode,
+	TreeFieldFromImplicitField,
+} from "@fluidframework/tree/internal";
+
+/**
+ * Given a tree node, generates a set of LLM friendly, unique ids for each node in a given Shared Tree.
+ * @remarks - simple id's are important for the LLM and this library to create and distinguish between different types certain TreeEdits
```

Comment:
```suggestion
 * @remarks Simple IDs are important for the LLM and this library to create and distinguish between different types of TreeEdits
```

Reply:
Updated in commit 6eb662389384cc2360c519307d3b33d3a7a02309